File: lab_12/message.cpp
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main() {
    string message;
    cout << "Enter your message: ";
    getline(cin, message);

    ofstream outFile("message.txt");
    outFile << message << endl;
    outFile.close();

    ifstream inFile("message.txt");
    if (inFile) {
        string line;
        cout << "Contents of message.txt:" << endl;
        while (getline(inFile, line)) {
            cout << line << endl;
        }
        inFile.close();
    } else {
        cout << "The file has no data." << endl;
    }

    return 0;
}

File: lab_12/message.py
message = input("Enter your message: ")

with open("message.txt", "w") as outFile:
    outFile.write(message + "\n")

try:
    with open("message.txt", "r") as inFile:
        print("Contents of message.txt:")
        for line in inFile:
            print(line, end="")
except FileNotFoundError:
    print("The file has no data.")
